frontend/src/screens/Start.tsx

```diff
@@ -24,10 +24,31 @@ export default function Start() {
   const [unlockPassword, setUnlockPassword] = React.useState("");
   const [loading, setLoading] = React.useState(false);
   const [buttonText, setButtonText] = React.useState("Login");
-  useInfo(true); // poll the info endpoint to auto-redirect when app is running
+  const [startupError, setStartupError] = React.useState("");
```

Comment:
this is just a string value, there is no need to use state for it, we can use it directly from `info.startupError`

Reply:
We need this state variable because of [this](https://github.com/getAlby/hub/pull/571#discussion_r1741494171)